.rubocop.ruby.yml

```
@@ -223,6 +223,10 @@ Style/Next:
 Style/NilComparison:
   Enabled: false
 
+Style/NumberedParametersLimit:
+  Enabled: true
+  Max: 1
+
```

Suggestion: also restrict to single line. [[docs](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/NumberedParameters)]


```suggestion
Style/NumberedParameters:
  Enabled: true
  EnforcedStyle: allow_single_line

Style/NumberedParametersLimit:
  Enabled: true
  Max: 1

```

---

+1 for the single-line restriction.